.claude/skills/linch-workflow/SKILL.md

```
@@ -38,18 +50,64 @@ Read the user's request and classify:
+
+Before creating a PR, request cross-model review for a second opinion.
+**Must ask user for approval before invoking external tools.**
+
+1. **Detect available tools** — check all known AI CLI tools:
+   ```bash
+   for cmd in codex gemini claude aider llm mods sgpt ollama fabric goose avante trae; do
+     which "$cmd" 2>/dev/null && echo "FOUND: $cmd"
+   done
+   ```
+   Known tools and their non-interactive modes:
+   | Tool | Backend | Non-interactive mode |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)